index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Portfolio</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+
+<body>
+  <nav class="sticky">
+    <div class="main-choose"><a href="#welcome-section">About</a></div>
+    <div class="main-choose"><a href="#projects">Work</a></div>
+    <div class="main-choose"><a href="#footer">Contact</a></div>
+  </nav>
```

Comment:
- [ ] Please use list `<ul>` or `<ol>` for the social links instead of `<div>.` Whenever we have a list of similar or identical elements, we should group them together on a list. It is also better for SEOs interpretation. 😉